### DISCUSSION TITLE
Acheter des diamants

### DISCUSSION THREAD

**Discussion by RayZZo9** [Crownicles/Crownicles#1894] [Suggestions fermées]
Voilà, j'ai eu l'idée de pouvoir acheter des diamants depuis le magasin ou le magasinmissions.

Evidemment il faudrait faire en sorte que l'on ne puisse pas faire du business avec, mais je pense que c'est une bonne idée, pas vous ?

Merci de votre attention.

**Comment by Bombafox** [comment]
On gagne déjà des 💎 dans la campagne et dans les missions journalière.
Perso j'en vois pas l'intérêt je trouve que c'est suffisant.

**Reply by BastLast** [reply]
Il parle de pouvoir acheter des diamants dans le magasin des mission donc je pense pas qu'il parle de gemmes sinon il serait en train de proposer d'acheter des gemmes avec des gemmes

**Reply by Nargacaura** [reply]
Théoriquement, je pense qu'il est *effectivement* en train de parler de gemmes en mentionnant "diamants", donc acheter des gemmes avec des gemmes... c'est pas un peu dépenser ses gemmes pour rien?
Sinon, je n'vois pas trop l'intérêt de mettre des gemmes à vendre dans le shop non plus.

**Reply by Pietagorh** [reply]
Nan mais les acheter depuis le magasin des missions avec des 💰 logiquement
Si il dit "ou magasin des missions" c'est sûrement parce que vu que c'est la monnaie du magasin des missions ce serait logique de les acheter là-bas

**Comment by BastLast** [comment]
À quoi serviraient les diamants ?

**Comment by RayZZo9** [comment]
Acheter des gemmes avec de l'argent
des gemmes désolé
pas des diamants

**Comment by BastLast** [comment]
Acheter des gemmes avec de l'argent n'est pas sur la roadmap

**Reply by QQtin3** [reply]
Ça a été pensé à l'origine pour faire une distinction entre les deux justement pour "forcer" en tout cas inciter les gens à faire des quêtes s'ils veulent des récompenses pas accessibles autrement
